cpp/src/slice2java/JavaUtil.cpp

```diff
@@ -343,83 +319,6 @@ Slice::JavaGenerator::output() const
     return *_out;
 }
 
-//
-// If the passed name is a scoped name, return the identical scoped name,
-// but with all components that are Java keywords replaced by
-// their "_"-prefixed version; otherwise, if the passed name is
-// not scoped, but a Java keyword, return the "_"-prefixed name;
-// otherwise, return the name unchanged.
-//
-string
-Slice::JavaGenerator::fixKwd(const string& name)
-{
-    if (name.empty())
-    {
-        return name;
-    }
-    if (name[0] != ':')
-    {
-        return lookupKwd(name);
-    }
-    vector<string> ids = splitScopedName(name);
-    transform(ids.begin(), ids.end(), ids.begin(), [](const string& id) -> string { return lookupKwd(id); });
-    stringstream result;
-    for (const auto& id : ids)
-    {
-        result << "::" + id;
-    }
-    return result.str();
-}
-
-string
-Slice::JavaGenerator::convertScopedName(const string& scoped, const string& prefix, const string& suffix)
```

Comment:
Now that `mappedScoped` has a 'separator' argument, we can let `libSlice` give us the separator we want (`.`),
instead of needing this function to massage it after-the-fact.